components/cluster/libstatgrab/libstatgrab.p5m

```diff
@@ -24,15 +25,21 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-file path=usr/bin/saidar
-file path=usr/bin/statgrab
-file path=usr/bin/statgrab-make-mrtg-config
-file path=usr/bin/statgrab-make-mrtg-index
+file path=usr/bin/$(MACH64)/saidar
```

Comment:
Please, set CONFIGURE_BINDIR.64 like [here](https://github.com/OpenIndiana/oi-userland/blob/b6ca80a20fb2f5e4e777be54f0bdf34372fde345/components/shell/dash/Makefile#L41)

Reply:
This will not work:
echo "CONFIGURE_BINDIR.64=/usr/bin/amd64"
We need a varaiable with the value "usr/bin/amd64".

Reply:
Ok, I got it.
I thought all 64bit binaries should go in usr/(s)bin/$MACH64).
But this seems to be wrong and if only 64bit binaries are created we use the the paths without $(MACH64).